tests/unit/test_schema.py

```diff
@@ -632,3 +676,54 @@ def test_valid_mapping_representation(self):
 
         result = self._call_fut(schema)
         self.assertEqual(result, expected_schema)
+
+
+class TestPolicyTags(unittest.TestCase):
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.bigquery.schema import PolicyTagList
+
+        return PolicyTagList
+
+    def _make_one(self, *args, **kw):
+        return self._get_target_class()(*args, **kw)
+
+    def test_constructor(self):
+        empty_policy_tags = self._make_one()
+        self.assertIsNotNone(empty_policy_tags.names)
+        self.assertEqual(len(empty_policy_tags.names), 0)
+        policy_tags = self._make_one(["foo", "bar"])
+        self.assertEqual(policy_tags.names, ["foo", "bar"])
+
+    def test_from_api_repr(self):
+        klass = self._get_target_class()
+        api_repr = {"names": ["foo"]}
+        policy_tags = klass.from_api_repr(api_repr)
+        self.assertEqual(policy_tags.to_api_repr(), api_repr)
```

Comment:
It seems that creating an instance from `None` case is missing? Let's add that, too.

Reply:
Done